test/functional/toolchains/cmake/cmakedeps2/test_cmakeconfigdeps_frameworks.py

```diff
@@ -63,10 +79,19 @@ class MyFramework(ConanFile):
         name = "frame"
         version = "1.0"
         settings = "os", "arch", "compiler", "build_type"
-        languages = ["C++"]
-        package_type = "{'static-library' if shared else 'shared-library'}"
```

Comment:
That was wrong, I know. I should be:
```python
package_type = "{'shared-library' if shared else 'static-library'}"
```
But even putting that, SHARED mode was giving me problems. I'm looking into that. Using the old-fashioned way is working fine